_inc/client/components/dash-item/style.scss

```diff
@@ -69,7 +69,7 @@
 }
 .jp-dash-item__active-label {
 	display: inline-block;
-	padding: rem( 7px ) 0;
```

Comment:
It was set this way to line up with the buttons (same size, weight, padding). Haven't tested your change yet, though.